### PR TITLE
K8SPG-260: Fix restart after upgrade

### PR DESCRIPTION
**Pull request by egegunes**
[K8SPG-260](https://jira.percona.com/browse/K8SPG-260) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
Pods get restarted after deploying 2.5.0 operator without changing `crVersion`.

**Cause:**
No version check in crunchy controllers.

**Solution:**
Add version checks.

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?
- [x] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [x] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [x] Is an E2E test/test case added for the new feature/change?
- [x] Are unit tests added where appropriate?

**Config/Logging/Testability**
- [x] Are all needed new/changed options added to default YAML files?
- [x] Did we add proper logging messages for operator actions?
- [x] Did we ensure compatibility with the previous version or cluster upgrade process?
- [x] Does the change support oldest and newest supported PG version?
- [x] Does the change support oldest and newest supported Kubernetes version?

[K8SPG-260]: https://perconadev.atlassian.net/browse/K8SPG-260?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by inelpandzic — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/custom-extensions.log) |
| custom-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/custom-tls.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/demand-backup.log) |
| finalizers | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/finalizers.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/init-deploy.log) |
| major-upgrade | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/major-upgrade.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/operator-self-healing.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/pitr.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/start-from-backup.log) |
| tablespaces | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/tablespaces.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/telemetry-transfer.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/upgrade-consistency.log) |
| upgrade-minor | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/upgrade-minor.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/users.log) |
| We run 19 out of 19|

commit: https://github.com/percona/percona-postgresql-operator/pull/918/commits/acc08156fe56fd249221ccd2dda8c5a012731ee0
image: `perconalab/percona-postgresql-operator:PR-918-acc08156f`

**Review by nmarukovich — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/custom-extensions.log) |
| custom-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/custom-tls.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/demand-backup.log) |
| finalizers | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/finalizers.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/init-deploy.log) |
| major-upgrade | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/major-upgrade.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/operator-self-healing.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/pitr.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/start-from-backup.log) |
| tablespaces | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/tablespaces.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/telemetry-transfer.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/upgrade-consistency.log) |
| upgrade-minor | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/upgrade-minor.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/acc08156f/users.log) |
| We run 19 out of 19|

commit: https://github.com/percona/percona-postgresql-operator/pull/918/commits/acc08156fe56fd249221ccd2dda8c5a012731ee0
image: `perconalab/percona-postgresql-operator:PR-918-acc08156f`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/custom-extensions.log) |
| custom-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/custom-tls.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/demand-backup.log) |
| finalizers | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/finalizers.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/init-deploy.log) |
| major-upgrade | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/major-upgrade.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/operator-self-healing.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/pitr.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/start-from-backup.log) |
| tablespaces | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/tablespaces.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/telemetry-transfer.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/upgrade-consistency.log) |
| upgrade-minor | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/upgrade-minor.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/899605488/users.log) |
| We run 19 out of 19|

commit: https://github.com/percona/percona-postgresql-operator/pull/918/commits/899605488617227146c113d7f6c45719f5616be0
image: `perconalab/percona-postgresql-operator:PR-918-899605488`

**Review by inelpandzic — APPROVED** [review]

**Review by inelpandzic — COMMENTED** [review]
@egegunes pls check these failing tests.

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/custom-extensions.log) |
| custom-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/custom-tls.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/demand-backup.log) |
| finalizers | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/finalizers.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/init-deploy.log) |
| major-upgrade | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/major-upgrade.log) |
| monitoring | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/operator-self-healing.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/pitr.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/start-from-backup.log) |
| tablespaces | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/tablespaces.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/telemetry-transfer.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/upgrade-consistency.log) |
| upgrade-minor | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/upgrade-minor.log) |
| users | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/users.log) |
| We run 19 out of 19|

commit: https://github.com/percona/percona-postgresql-operator/pull/918/commits/419808095eb3dad83ba3ca41cd12e4c1fa86da94
image: `perconalab/percona-postgresql-operator:PR-918-419808095`

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/custom-extensions.log) |
| custom-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/custom-tls.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/demand-backup.log) |
| finalizers | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/finalizers.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/init-deploy.log) |
| major-upgrade | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/major-upgrade.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/operator-self-healing.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/pitr.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/start-from-backup.log) |
| tablespaces | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/tablespaces.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/telemetry-transfer.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/upgrade-consistency.log) |
| upgrade-minor | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/upgrade-minor.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/419808095/users.log) |
| We run 19 out of 19|

commit: https://github.com/percona/percona-postgresql-operator/pull/918/commits/419808095eb3dad83ba3ca41cd12e4c1fa86da94
image: `perconalab/percona-postgresql-operator:PR-918-419808095`

**Review by hors — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/custom-extensions.log) |
| custom-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/custom-tls.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/demand-backup.log) |
| finalizers | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/finalizers.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/init-deploy.log) |
| major-upgrade | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/major-upgrade.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/operator-self-healing.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/pitr.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/start-from-backup.log) |
| tablespaces | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/tablespaces.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/telemetry-transfer.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/upgrade-consistency.log) |
| upgrade-minor | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/upgrade-minor.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/users.log) |
| We run 19 out of 19|

commit: https://github.com/percona/percona-postgresql-operator/pull/918/commits/522c655e6891794075cc468b309de98d726c3d7d
image: `perconalab/percona-postgresql-operator:PR-918-522c655e6`

File: pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go [hunk 1]
```diff
@@ -733,3 +734,13 @@ func NewPostgresCluster() *PostgresCluster {
 	cluster.SetGroupVersionKind(GroupVersion.WithKind("PostgresCluster"))
 	return cluster
 }
+
+const LabelVersion = "pgv2.percona.com/version"
```
Comment (inelpandzic) [inline pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go:738]: Can you reuse this: https://github.com/percona/percona-postgresql-operator/blob/34f1bd17199d2fa7fe73643230b8f58c621a88d4/internal/naming/labels.go#L28

**Review by inelpandzic — APPROVED** [review]

File: pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go (hunk 1 above)
Comment (egegunes) [reply pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go:738]: unfortunately, no. it causes cyclical import

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/custom-extensions.log) |
| custom-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/custom-tls.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/demand-backup.log) |
| finalizers | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/finalizers.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/init-deploy.log) |
| major-upgrade | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/major-upgrade.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/operator-self-healing.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/pitr.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/start-from-backup.log) |
| tablespaces | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/tablespaces.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/telemetry-transfer.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/upgrade-consistency.log) |
| upgrade-minor | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/upgrade-minor.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-918/522c655e6/users.log) |
| We run 19 out of 19|

commit: https://github.com/percona/percona-postgresql-operator/pull/918/commits/522c655e6891794075cc468b309de98d726c3d7d
image: `perconalab/percona-postgresql-operator:PR-918-522c655e6`

**Review by pooknull — APPROVED** [review]

**Pull request merged**